Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/largefile/CONNECTTimestamp.java

```diff
@@ -87,38 +86,55 @@ public boolean isExpired(Date invocationDate) {
      * @return true if the timestamp is before (now-timeToLive), false otherwise
      */
     public boolean verifyCreated(int timeToLive, int futureTimeToLive, Date invocationDate) {
-        if (invocationDate == null) {
-            invocationDate = new Date();
-        }
+        Date currentInvocationDate = invocationDate == null ? new Date() : invocationDate;
 
-        long invocationTime = invocationDate.getTime();
+        long invocationTime = currentInvocationDate.getTime();
         if (futureTimeToLive > 0) {
-            invocationDate.setTime(invocationTime + futureTimeToLive * 1000);
+            currentInvocationDate.setTime(invocationTime + futureTimeToLive * 1000);
         }
+
         // Check to see if the created time is in the future
-        if (createdDate != null && createdDate.after(invocationDate)) {
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Validation of Timestamp: The message was created in the future!");
-            }
+        if (!verifyCreatedAfter(currentInvocationDate)) {
             return false;
         }
 
         // Calculate the time that is allowed for the message to travel
         invocationTime -= timeToLive * 1000;
-        invocationDate.setTime(invocationTime);
+        currentInvocationDate.setTime(invocationTime);
 
         // Validate the time it took the message to travel
-        if (createdDate != null && createdDate.before(invocationDate)) {
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("Validation of Timestamp: The message was created too long ago");
-            }
+        if (!verifyCreatedBefore(currentInvocationDate)) {
             return false;
         }
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("Validation of Timestamp: Everything is ok");
-        }
+        LOG.info("Validation of Timestamp: Everything is ok");
         return true;
     }
 
+    /**
+     * Checks if created time is in the future
+     */
+    private boolean verifyCreatedAfter(Date invocationDate) {
+        // Check to see if the created time is in the future
+        if (getCreated() != null && getCreated().after(invocationDate)) {
+            LOG.info("Validation of Timestamp: The message was created in the future!");
```

Comment:
This is a self explanatory method. Do we need log statement here? Also try to make use of Java ternary operator